cmd/kubestellar-where-resolver/cmd/where-resolver-cmd.go

```diff
@@ -83,39 +83,43 @@ func Run(ctx context.Context, options *resolveroptions.Options) error {
 	logger := klog.Background()
 	ctx = klog.NewContext(ctx, logger)
 
-	// create edgeSharedInformerFactory
-	espwRestConfig, err := options.EspwClientOpts.ToRESTConfig()
+	spaceManagementConfig, err := options.SpaceMgtOpts.ToRESTConfig()
 	if err != nil {
-		logger.Error(err, "failed to create config from flags")
+		logger.Error(err, "Failed to create space management config from flags")
 		return err
 	}
-	edgeClient, err := edgeclientset.NewForConfig(espwRestConfig)
+	spaceClient, err := spaceclient.NewMultiSpace(ctx, spaceManagementConfig)
 	if err != nil {
-		logger.Error(err, "failed to create clientset for KubeStellar Core Space")
+		logger.Error(err, "Failed to create space-aware client")
 		return err
 	}
-	edgeSharedInformerFactory := edgeinformers.NewSharedScopedInformerFactoryWithOptions(edgeClient, resyncPeriod)
+	spaceProviderNs := spacemanager.ProviderNS(options.SpaceProvider)
 
-	// create where-resolver
-	baseRestConfig, err := options.BaseClientOpts.ToRESTConfig()
+	coreRestConfig, err := spaceClient.ConfigForSpace(options.KcsName, spaceProviderNs)
```

Comment:
"core" is too vague here, this is the RestConfig for the "KubeStellar Core Space". This is replacing "espwRestConfig", should be named "kcsRestConfig" (if renamed at all).

Reply:
changed to "kcsRestConfig"

Reply:
done